action.php

```diff
@@ -179,6 +179,14 @@ protected function collectExportPages(Doku_Event $event) {
             $dir = utf8_encodeFN(str_replace(':', '/', $pdfnamespace));
             search($result, $conf['datadir'], 'search_allpages', $opts, $dir);
 
+            // exclude ids
+            $excludes = $INPUT->arr('excludes');
+            if (!empty($excludes)) {
+                $result = array_filter(array_map(function ($item) use ($excludes) {
+                    return array_search($item['id'], $excludes) === false ? $item : '';
+                }, $result));
```

Comment:
I think the `array_map` is unnecessary. `array_filter` accepts a function that returns true or false to decide which item to keep. That's what you're doing here.

Reply:
The redundant function is gone.